MTMImporter/Importer.cs

```diff
@@ -318,21 +320,32 @@ public static void Main(string[] args)
         {
             try
             {
-                ValidateArguments(args);
-                var uri = args[0];
-                var username = args[1];
-                var password = args[2];
-                var domain = args[3];
-                var teamProject = args[4];
-                Parser parser = new Parser();
-                var resultsSession = parser.Parse(args[5]);
-                var testRun = ConvertResultToTestRun(resultsSession);
-                var importer = new Importer(uri, username, password, domain);
-                importer.Import(teamProject, testRun);
+                bool isEncrypt = args.Length == 2 && args[0] == "--encrypt";
+                if (isEncrypt)
+                {
+                    Console.WriteLine(SimpleCipher.Encrypt(args[1]));
+                }
+                else
+                {
+                    var options = new Options();
+                    if (!CommandLine.Parser.Default.ParseArguments(args, options))
+                    {
+                        Environment.Exit(CommandLine.Parser.DefaultExitCodeFail);
+                    }
+
+                    var password = SimpleCipher.Decrypt(options.Password);

```

Comment:
What if the user passes a plain-text password? Looks like it won't be supported anymore. The SimpleEncryptor should probably be refactored into a TryDecrypt method so you can fall back to the passed value instead of throwing an exception.